pom.xml

```diff
@@ -142,6 +142,11 @@
       <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I got some errors about `junit-jupiter-params` being used but not declared e.g. `EnumNamingStrategiesTest` - do we need a dependency here?

Reply:
Let me check and possible merge separately

Reply:
Merged. Guessing it came as transitive dependency, but module-info.java declared requirement anyway.

Reply:
Thanks!